Models/Employee.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DFBangazon.Models
+{
+    public class Employee
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public bool IsSupervisor { get; set; }
+        public int DepartmentId { get; set;}
+        public Department Department { get; set; }
+        public List<EmployeeComputer> EmployeeComputer { get; set; }
```

Comment:
Do not need a list of computers, just need the one single current computer.
Also do not need list of trainings.


No lists for employee.